state.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  backend "azurerm" {}
```

Comment:
this is what adds saving the state to azure, if it's not here then the state won't get saved to azure ever.

note that the state file is keyed by the 'product' var in your Jenkinsfile so you might want to change it to be more specific to your repo:

https://github.com/hmcts/ccpay-payment-api-gateway/blob/master/Jenkinsfile_CNP#L5

Reply:
Looks like the state config never existed before. But the last time the pipeline executed successfully on 12/10/2020, it was pointing at the state file in question now.

Will discuss with project team to change the product name. However, we need to import the existing resource.

Reply:
Updated the Jenkins file with new name.